### PR TITLE
Added the file watchers config to the environment

### PR DESCRIPTION
**Pull request by nemanja-da**

### CODE REVIEW

File: environments/daml-sdk-env/build/1_installprerequisites.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Configuring the environment for more file watchers
+echo "fs.inotify.max_user_watches=524288" >> /etc/sysctl.conf
+
```
Comment (ghost) [inline environments/daml-sdk-env/build/1_installprerequisites.sh:5]: I think you should do a `sudo sysctl -p` here

Comment (nemanja-da) [reply environments/daml-sdk-env/build/1_installprerequisites.sh:5]: Would this work better? Updated the line

Comment (ghost) [reply environments/daml-sdk-env/build/1_installprerequisites.sh:5]: not sure what's better about the `tee` but the important thing is the `sysctl -p`. So this should work.

**Review by ghost — APPROVED** [review]
Thanks!

**Pull request merged**